docs/source/api/plugin/usage-reporting.md

```diff
@@ -423,8 +423,8 @@ Specify this function to create a signature for a query. This option is not reco
 |--------|-------------|
 | `{ none: true }` | If you provide this object, no GraphQL variable values are sent to Apollo Studio. This is the default behavior. |
 | `{ all: true }` |  If you provide this object, **all** GraphQL variable values are sent to Apollo Studio. |
-| `{ onlyNames: ["apple", "orange"]}`| If you provide an object with this structure, only values of the GraphQL variables with names that appear in the array are sent to Apollo Studio. Case-sensitive. |
-| `{ exceptNames: ["apple", "orange"]}`| If you provide an object with this structure, all GraphQL variable values **except** values of variables with names that appear in the array are sent to Apollo Studio. Case-sensitive. |
+| `{ onlyNames: ["apple", "orange"]}`| If you provide an object with this structure, only values of the GraphQL variables with names that appear in the array are sent to Apollo Studio. Use the `transform` function below if you'd like only to include _specific_ values from your GraphQL variables. Case-sensitive. |
```

Comment:
I might change both of these to the more generic:

"To filter individual fields of a variable that contains an input type, use the `transform` function below instead."

(Sorry to keep spinning tires on this 😄, we just had another forum question about this topic and I think this tweak might help)

Reply:
Thank you for the suggestion!